### DISCUSSION TITLE
How to become an official maintainer/contributor?

### DISCUSSION THREAD

**Discussion by Hamzakh777** [web-infra-dev/rspack#3391] [General]
Hello, 

I have been following and using this project since its early days, I always didn't like the idea of re-inventing the bundlers (Vite, esbuilt,...) when we could simply create another Webpack using Rust that has 100% interoperability with all the plugins and amazing ecosystem that exists today. 

I have taken a look at the code, forked the project locally, and looked at the open issues on this repo. But I got lost on where to even begin, who to communicate with, how to make sure the issue I want to pick is not picked by someone else, low priority, etc, etc...

That's why I want to create this discussion. I want to get a better idea from the team behind this on how to start contributing on a regular basis and also any point of contact that could make this process easier (happy to jump on calls, join a Slack channel, or whatever medium is preferred).

Thanks!

**Comment by hardfist** [comment]
sorry about this, we are working on https://web-infra-dev.github.io/rspack-dev-guide/ which maybe helpful to you，and we have good first issue label for new contributors，you can pick any one if it is not assigned to anyone or no work for long time，we have discord channel for communication https://discord.gg/79ZZ66GH9E where you can ask any questions or you can DM me or other team member about private rspack questions

**Reply by Hamzakh777** [reply]
Thank you very much for the reply @hardfist, I'll give this a go!

**Comment by hyf0** [comment]
Thanks for your interest in this project. Making contributions would help of becoming a collaborator。

@Boshen is drafting the project governance of Rspack. You'll see it soon.
